src/main/java/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -60,6 +60,9 @@
   private boolean m_needToRecordBattleStatistics = true;
   private boolean m_needToCheckDefendingPlanesCanLand = true;
   private boolean m_needToCleanup = true;
+  private boolean needToFindRocketTargets = true;
+  private boolean needToFireRockets = true;
```

Comment:
Ugh. You didn't like RocketHelper being too stateful before, now you're saying to make it more stateful. Am I missing something?